README.md

```diff
@@ -1 +1,2 @@
-# friendship-quiz
\ No newline at end of file
+# friendship quiz 
+Alex
```

Comment:
Hi Alex, your code in friendship-quiz.js looks awesome! I ***loved*** your use of a for-loop to iterate through `answerList`, and your use of a boolean variable (`found`) to hold whether or not `guess` has been found in `answerList` yet. 
I have two comments:
- `console.log()` is a super helpful function to use when you're writing your code, but the user actually won't be interacting with the browser console. This would be a good line to delete when thinking about what lines of your code are unnecessary for the final product!
- It would've been nice to add a helpful comment about what the `found` variable is doing!

Great work 😎